src/test/java/com/exasol/adapter/dialects/db2/DB2SqlDialectTest.java

```diff
@@ -92,7 +92,7 @@ void testValidateCatalogProperty() {
         final PropertyValidationException exception = assertThrows(PropertyValidationException.class,
                 sqlDialect::validateProperties);
         MatcherAssert.assertThat(exception.getMessage(), containsString(
-                "The dialect DB2 does not support CATALOG_NAME property. Please, do not set the CATALOG_NAME property."));
+                "The dialect DB2 does not support CATALOG_NAME property. Please, do not set the \"CATALOG_NAME\" property."));
```

Comment:
```suggestion
                "The dialect DB2 does not support \"CATALOG_NAME\" property. Please, do not set it."));
```

Reply:
This won't work, because it requires changes in the common part again